<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winPreRes</class>
 <widget class="QWidget" name="winPreRes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Codigo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="L_cod"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pregunta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="L_nom"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Puntos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="L_pun"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Categoria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Tipos de datos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ordenamiento</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Busqueda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TDA Estática</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TDA Dinámicos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Almacenamiento secundario</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descripcion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Correcta</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="B_Salir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="B_Capturar">
       <property name="text">
        <string>Capturar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
